src/index.js

```diff
@@ -2,16 +2,20 @@ import React from 'react';
 import { Provider } from 'react-redux';
 import * as ReactDOM from 'react-dom';
 import 'abortcontroller-polyfill/dist/polyfill-patch-fetch';
-
 import 'vis-network/dist/vis-network.css';
 import 'react-toastify/dist/ReactToastify.css';
 import 'sagas';
 import { App } from 'components/App/App';
 import { store, history } from 'sagas';
 import { GlobalStyle } from 'globalStyles';
-
 import { ConnectedRouter } from 'connected-react-router';
 
+// The app flow is following:
+// 1) The global code is executed. So if you have for example console.log() at the root of file it will execute
```

Comment:
What do you mean by "global code"? You mean code in `index.js` is executed?

Reply:
Code in files that is executed. So for example we have `file.js` and inside it first line is
```
console.log(...)
```
It will be executed immediately on js load

Reply:
aha, so here
>So if you have for example console.log() at the root of file it will execute

you mean "at the root of any file" ?
I thought you mean just this file - `index.js`.

By the way I think it's called `top` of file, not `root` :) Not sure there is such a concept as "root of file".

Reply:
Yes. Totally agree, perhaphs should change explanation on the one that wrote here